### PR TITLE
To use IISPlatformHandler instead of Microsoft.AspNet.Server.IIS

### PR DESCRIPTION
**Pull request by Kagamine**
Use IISPlatformHandler instead Microsoft.AspNet.Server.IIS

### CODE REVIEW

**Comment by Kagamine** [comment]
it seems something wrong with https://github.com/aspnet/IISIntegration

File: test/WebSites/AddServicesWebSite/Startup.cs [hunk 1]
```diff
@@ -9,6 +9,8 @@ public class Startup
     {
         public void Configure(IApplicationBuilder app)
         {
+            app.UseIISPlatformHandler();
+
```
Comment (Eilon) [inline test/WebSites/AddServicesWebSite/Startup.cs]: Need to trim trailing whitespace (might need to check other files too).

Comment (Kagamine) [inline test/WebSites/AddServicesWebSite/Startup.cs]: sorry I didn't understand what you mean. Should I remove the white line? But there are whitelines also in the files in each app.usexxx

Comment (Eilon) [inline test/WebSites/AddServicesWebSite/Startup.cs]: @Kagamine sorry line 13 has 12 spaces on it (indentation) - they just need to be trimmed. Keep the empty line, though.

**Comment by Eilon** [comment]
Also please squash the commits when you have a chance, thanks!

**Pull request closed without merging**